frontend/src/SignInPage.js

```diff
@@ -15,90 +18,93 @@ import {
   useToast,
 } from '@chakra-ui/core'
 import { Link as RouteLink, useHistory } from 'react-router-dom'
-import { useMutation, useApolloClient } from '@apollo/react-hooks'
+import { useMutation } from '@apollo/react-hooks'
 import gql from 'graphql-tag'
 import { Formik, Field } from 'formik'
 
 export function SignInPage() {
-  const [show, setShow] = React.useState(false)
-  const handleClick = () => setShow(!show)
-
-  const client = useApolloClient()
   const history = useHistory()
 
   const toast = useToast()
 
-  const [signIn, { loading, error, data }] = useMutation(gql`
-    mutation SignIn($userName: EmailAddress!, $password: String!) {
-      signIn(userName: $userName, password: $password) {
-        user {
-          userName
-          failedLoginAttempts
-          tfaValidated
+  const { i18n } = useLingui()
+
+  const [signIn, { loading, error }] = useMutation(
+    gql`
+      mutation SignIn($userName: EmailAddress!, $password: String!) {
+        signIn(userName: $userName, password: $password) {
+          user {
+            userName
+            failedLoginAttempts
```

Comment:
Hey Guys, was just taking a quick peak at this PR, and the user object has been updated and you can no longer access the `failedLoginAttempts` field for user

Reply:
Thanks @IdezHD,  I do not think it is actually being used for any functionality right now.  I will double check with Mike and let him update the PR! :smile:

Reply:
Ahh, nice catch. Lemme take care of that.